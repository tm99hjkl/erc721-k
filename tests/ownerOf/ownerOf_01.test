makeAccount(1)
setCaller(1)
makeNFT(999, 1)
ownerOf(999)
