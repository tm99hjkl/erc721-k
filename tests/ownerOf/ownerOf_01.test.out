<ERC721>
  <k>
    1 ~> .
  </k>
  <caller>
    1
  </caller>
  <accounts>
    <account>
      <accountID>
        1
      </accountID>
      <balance>
        1
      </balance>
      <operate>
        1 |-> true
      </operate>
    </account>
  </accounts>
  <nfts>
    <nft>
      <nftID>
        999
      </nftID>
      <nftOwner>
        1
      </nftOwner>
      <nftController>
        1
      </nftController>
    </nft>
  </nfts>
  <log>
    Event: 
  </log>
</ERC721>
